static/js/show-worklist.js

```diff
@@ -1,3 +1,11 @@
+var worklists = [];
+var worklist_names = [];
+var usernames = [];
+var input_by_worklist_name;
+var input_by_username;
+var awesomplete_by_worklist_name;
+var awesomplete_by_username;
+
```

Comment:
Move the uses of these variables to this file too, please. It's pretty odd to see the variables declared here but only actually used in inline script in the HTML file.

Reply:
Done.